traffic_ops/install/bin/install_goose.sh

```diff
@@ -14,11 +14,13 @@
 # limitations under the License.
 #
 
-echo "Now installing goose"
-export GOPATH=/opt/traffic_ops/go
-mkdir -p $GOPATH
+trap "echo Error on line ${LINENO} of '$0'; exit 1" ERR;
+set -o errexit -o nounset;
+echo "Now installing goose";
+export GOPATH=/opt/traffic_ops/go;
+mkdir -p $GOPATH;
 
-go get bitbucket.org/liamstask/goose/cmd/goose
-go get github.com/lib/pq
+go get bitbucket.org/liamstask/goose/cmd/goose;
+go get github.com/lib/pq;
 
-echo "Successfully installed goose to $GOPATH/bin/goose"
+echo "Successfully installed goose to $GOPATH/bin/goose";
```

Comment:
why the `;`?

Reply:
Why generally? I personally find semicolons nice if I am copying and pasting an entire script into a terminal, because if the TTY misses a newline, it still works. Since that is not related to this PR though, I can remove them.

Reply:
Removed in 6d3edb9c66

Reply:
Yeah, I was just curious why generally since I haven't seen that as a style before.